### PR TITLE
Align tcti tools

### PR DESCRIPTION
**Pull request by flihp**

### CODE REVIEW

**Comment by flihp** [comment]
this resolves #330

File: src/tabrmd.c [hunk 1]
```diff
@@ -265,6 +265,58 @@ show_version (const gchar  *option_name,
...
+ * if no semicolon is present in the combined_conf string it will be assumed
+ * that the whole string is the 'name'. To provide a 'conf' string to the
+ * default TCTI the first character of the combined_conf string must be a
+ * semicolon.
+ * If a field in the combined_conf string indicates a default value then
+ * the provided tcti_filename or tcti_conf will not be set. This is to allow
+ * defaults to be set by the caller and only updated here if we're changing
+ * them.
+ */
+int
```
Comment (williamcroberts) [inline src/tabrmd.c]: bool return? You only use 0 or -1 for return.

File: src/tabrmd.c [hunk 2]
```diff
@@ -265,6 +265,58 @@ show_version (const gchar  *option_name,
...
+    }
+
+    split = strchr (combined_conf, ':');
+    /* no semicolon, combined_conf is tcti name */
+    if (split == NULL) {
+        *tcti_filename = combined_conf;
+        return 0;
+    }
+    split [0] = '\0';
+    if (combined_conf[0] != '\0') {
```
Comment (williamcroberts) [inline src/tabrmd.c]: I think this is dead code. if combined_conf is the empty string, wont line ```strchr()``` on line 304 return NULL and then bail on line 308?

Comment (flihp) [reply src/tabrmd.c]: That's what will happen if `combined_conf` is the empty string. But if `combined_conf` is `:` setting `split[0] = '\0'` will cause `combined_conf[0]` to be `'\0` and this conditional will ensure that we don't set `tcti_filename` thus preserving the default.

File: src/tabrmd.c (hunk 1 above)
Comment (flihp) [reply src/tabrmd.c]: sure

**Comment by flihp** [comment]
Think i got your two comments covered @williamcroberts. If not just make an issue and assign it to me.

File: man/tpm2-abrmd.8.in [hunk 3]
```diff
@@ -70,6 +78,29 @@ behavior.
 .TP
 \fB\-v,\ \-\-version\fR
 Disply version string.
+.SH EXAMPLES
+.TP 3
+Execute daemon with default TCTI and options:
+.B tpm2-abrmd
+.TP
+Execute daemon with default TCTI and provided config string:
+.B tpm2-abrmd --tcti=":/dev/tpm0"
```
Comment (williamcroberts) [inline man/tpm2-abrmd.8.in]: Ahh We don't support this in tools, this harkens back to the "dead-code" comment.

**Pull request merged**